src/schema/slack/types/mod.ts

```diff
@@ -7,6 +7,7 @@ const SlackPrimitiveTypes = {
   blocks: "slack#/types/blocks",
   oauth2: "slack#/types/credential/oauth2",
   rich_text: "slack#/types/rich_text",
+  message_ts: "string",
```

Comment:
From the doc it looks like we should use `slack#/types/message_ts` type instead of plain string? @shapirone?